test/src/test/java/hudson/PluginManagerTest.java

```diff
@@ -194,7 +194,7 @@ public void startPlugin(PluginWrapper plugin) throws Exception {
                     // plugins should be already visible in the UberClassLoader
                     assertFalse(activePlugins.isEmpty());
 
-                    uberClassLoader.loadClass("hudson.plugins.tasks.Messages");
+                    assertNotNull(uberClassLoader.loadClass("htmlpublisher.HtmlPublisher"));
```

Comment:
I unintentionally broke this test in commit 1b2ca8a709 by switching from `tasks.jpi` to `htmlpublisher.jpi` without updating the corresponding class name, but we never noticed because the return value is ignored. I fixed the return value problem by asserting that the value is not null, which would have exposed the original problem in the first place.